Mathlib/Analysis/SpecialFunctions/NonIntegrable.lean

```diff
@@ -92,7 +92,7 @@ theorem not_integrableOn_of_tendsto_norm_atTop_of_deriv_isBigO_filter_aux
     _ ≤ ∫ x in Ι c d, C * ‖g x‖ :=
       set_integral_mono_on hfi.norm.def (hgi.mono_set hsub') measurableSet_uIoc hg
     _ ≤ ∫ x in k, C * ‖g x‖ := by
-      apply set_integral_mono_set hgi
+      exact set_integral_mono_set hgi
```

Comment:
this has nothing to do with the PR (and there are situations where `apply foo` is faster than `exact foo` even when the latter does work)